tailwind.config.js

```diff
@@ -4,6 +4,19 @@ module.exports = {
     './components/**/*.{js,ts,jsx,tsx,md}',
   ],
   darkMode: "class",
+  safelist: [
+    'h-8',
+    'w-8',
+    'h-11',
+    'w-11',
+    'bg-slate-50',
+    'rounded-full',
+    '-ml-3',
+    'bg-cover',
+    'bg-center',
+    'border-2',
+    'border-white',
+  ],
```

Comment:
Are we sure we need to add this to the safelist? I think we can avoid this.

Reply:
I have removed the safelist from the tailwindconfig now 
Its working fine!